extensions/cornerstone/src/commandsModule.ts

```diff
@@ -604,6 +609,36 @@ function commandsModule({
       );
       toolGroup.setToolEnabled(ReferenceLinesTool.toolName);
     },
+    setDisplaySetToViewport: ({ displaySet }) => {
+      const cachedDisplaySetKeys = [
+        ...displaySetService.getDisplaySetCache().keys(),
+      ];
+      const displaySetKey = Object.keys(displaySet)[0];
+
+      // Check to see if computed display set is already in cache
+      if (!cachedDisplaySetKeys.includes(displaySetKey)) {
+        displaySetService._addDisplaySetsToCache([displaySet[displaySetKey]]);
+      }
+
+      // Get all viewports and their corresponding indexs
+      const { viewports } = viewportGridService.getState();
+      const viewportsToUpdate = [];
+
+      viewports.forEach((viewport, index) => {
+        viewportsToUpdate.push({
+          viewportIndex: index,
+          displaySetInstanceUIDs: [displaySetKey],
+          viewportOptions: {
+            initialImageOptions: viewport.viewportOptions.initialImageOptions,
```

Comment:
grab the viewport.viewportOptions above sot that you don't repeat viewport.viewportOptions three times here

Reply:
The viewport options for orientation change with each viewport so I don't believe I can do that while using .map()